utils/hare-bootstrap

```diff
@@ -288,39 +293,41 @@ jq '[.[] | {key, value: (.value | @base64)}]' < $conf_dir/consul-kv.json |
     consul kv import - > /dev/null
 echo ' OK'
 
-say 'Starting Consul on other nodes...'
-pids=()
-while read node bind_ip; do
-    ssh $node "PATH=$PATH $(which mk-consul-env) \
-                          --mode server --bind $bind_ip --join $join_ip &&
-               sudo systemctl start hare-consul-agent" &
+if [ "$skip_consul_start" = false ] ; then
```

Comment:
Now the question is if for some reason if the running Consul agent was stopped then who would restart it?
I would suggest the following,
- Continue iterating over all the nodes
- check if Consul is already running, if not start the Consul agent.

I think we also need to do this for node-start.